vscode-extensions/tls-sync-vscode-shared/src/config.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as vscode from 'vscode';
+
+import type { ICertificateManagerOptions } from '@rushstack/debug-certificate-manager';
+import { SETTINGS_PREFIX } from './constants';
+
+export interface IExtensionConfig extends ICertificateManagerOptions {
+  autoSync: boolean;
+}
+
+export function getConfig(outputChannel: vscode.OutputChannel): IExtensionConfig {
```

Comment:
It seems like the purpose of the `outputChannel` is purely for logging. Should logging for this be done here, or elsewhere? If here, should it take an `ITerminal` instead? Since you now have the `VSCodeOutputChannelTerminalProvider`. Also, since it's only logging, make the terminal/channel optional if it's left in here.